README.md

```diff
@@ -76,7 +76,7 @@ Your output could be something like:
 It takes 5 entries for 3 happy days to occur
 ```
 
-**think**: What are you going to do if there aren't at least 3 happy days? Where do you need to handle that case?
+**think**: What are you going to do if there aren't at least 3 happy days? Where do you need to handle that case? What am I going to do? Well, I would need to change my code to account for this. I would add an else to my if/elsif loop: puts "There were less than 3 happy days in #{entries} entries. :-(" (I actually did do this, because it made sense to me to do so).
```

Comment:
This is also true. But you should think about the fact that you are now returning two different types of data potentially. (I can explain this better in person)